snuba/datasets/storages/events_common.py

```diff
@@ -27,6 +28,19 @@
 from snuba.query.processors.prewhere import PrewhereProcessor
 from snuba.web.split import ColumnSplitQueryStrategy, TimeSplitQueryStrategy
 
+# This obnoxious amount backslashes is sadly required to escape the tag keys.
+# replaceRegexpAll(k, '(\\\\=|\\\\\\\\)', '\\\\\\\\\\\\1')
+# means running this on Clickhouse:
+# replaceRegexpAll(k, '(\\=|\\\\)', '\\\\\\1')
+# The (\\=|\\\\) pattern should be actually this: (|\=|\\). The additional
+# backslashes are because of Clickhouse escaping.
+TAGS_HASH_MAP_COLUMN = (
+    "arrayMap((k, v) -> cityHash64(concat("
+    "replaceRegexpAll(k, '(\\\\=|\\\\\\\\)', '\\\\\\\\\\\\1'), '=', v)), "
```

Comment:
While relay excludes tags that have the character `=` in the tag name, relying on that never to change for the correctness of this system seems pretty risky especially considering how hard it would be to trouble shoot the content of this column (hashed).
So we escape the tag name replacing `=` with `\=` and `\` with `\\`.

We do not need to escape the value, since we are not concatenating anything at the end of the value.